mudur/userprofile/migrations/0008_auto_20230731_0131.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2023-07-31 03:41
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('userprofile', '0007_auto_20220811_2310'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userprofile',
+            name='mobilephonenumber',
+            field=models.CharField(blank=True, max_length=14, verbose_name='Mobile Phone Number'),
```

Comment:
2 fields. mobilephonenumber and mobilephoneverify. 
mobilephonenumber can be empty or filled.
first check integer mobilephoneverify 0 or 1 
if mobilephoneverify 1 mobilephonenumber is filled if 0 empty. 
storing values in this way within database is NOT a good practice.